src/components/SubclassCheckbox.vue

```diff
@@ -5,7 +5,7 @@
 			type="checkbox"
 			id="include-subclasses"
```

Comment:
Now all these checkboxes have the same id, which means if I have two conditions and I click the label of the checkbox in the second condition, then the checkbox in the first condition is toggled.

Reply:
Something like `:id="id"`
and 
```js
data() {
    return {
        id: `include-subclasses-${Math.random()}`,
    };
},
```
should be good enough

Reply:
addressed!